python/ray/node.py

```diff
@@ -299,6 +299,9 @@ def _init_temp(self, redis_client):
         self._resource_dir = os.path.join(self._session_dir,
                                           "runtime_resources")
         try_to_create_directory(self._resource_dir)
+        # Create a directory to be used for container pidfile.
+        try_to_create_directory(
+            os.path.join(self._session_dir, "worker_container"))
```

Comment:
do you mind putting them into runtime_resources? This is the place where all runtime env related things being put.